<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>temp_hum_display</class>
 <widget class="QWidget" name="temp_hum_display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>温湿度检测</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 0, 127);
color: rgb(255, 255, 255);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="humidity_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 85, 0);
color: rgb(255, 255, 255);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="temperature_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 85, 127);
color: rgb(255, 255, 255);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 255);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">
color: rgb(255, 255, 255);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;</string>
          </property>
          <property name="text">
           <string>筛选:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="temp_hum_screen_comboBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>过去一分钟</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>过去一小时</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>过去一天</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="temp_hum_tableWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
